docs/world api.md

```diff
@@ -259,215 +262,124 @@ class MyGameWorld(World):
+    def __init__(self, player: int, name="", code=None, parent=None) -> None:
         super(MyGameLocation, self).__init__(player, name, code, parent)
         self.event = code is None
 ```
-in your `__init__.py` or your `locations.py`.
-
-### Options
-
-By convention options are defined in `options.py` and will be used when parsing
-the players' yaml files.
-
-Each option has its own class, inherits from a base option type, has a docstring 
-to describe it and a `display_name` property for display on the website and in
-spoiler logs.
-
-The actual name as used in the yaml is defined via the field names of a `dataclass` that is
-assigned to the world under `self.options_dataclass`. By convention, the strings
-that define your option names should be in `snake_case`.
-
-Common option types are `Toggle`, `DefaultOnToggle`, `Choice`, `Range`.
-For more see `Options.py` in AP's base directory.
-
-#### Toggle, DefaultOnToggle
-
-These don't need any additional properties defined. After parsing the option,
-its `value` will either be True or False.
 
-#### Range
-
-Define properties `range_start`, `range_end` and `default`. Ranges will be
-displayed as sliders on the website and can be set to random in the yaml.
-
-#### Choice
-
-Choices are like toggles, but have more options than just True and False.
-Define a property `option_<name> = <number>` per selectable value and
-`default = <number>` to set the default selection. Aliases can be set by
-defining a property `alias_<name> = <same number>`.
-
-```python
-option_off = 0
-option_on = 1
-option_some = 2
-alias_disabled = 0
-alias_enabled = 1
-default = 0
-```
-
-#### Sample
-```python
-# options.py
-
-from dataclasses import dataclass
-from Options import Toggle, Range, Choice, PerGameCommonOptions
-
-class Difficulty(Choice):
-    """Sets overall game difficulty."""
-    display_name = "Difficulty"
-    option_easy = 0
-    option_normal = 1
-    option_hard = 2
-    alias_beginner = 0  # same as easy
-    alias_expert = 2  # same as hard
-    default = 1  # default to normal
-
-class FinalBossHP(Range):
-    """Sets the HP of the final boss"""
-    display_name = "Final Boss HP"
-    range_start = 100
-    range_end = 10000
-    default = 2000
-
-class FixXYZGlitch(Toggle):
-    """Fixes ABC when you do XYZ"""
-    display_name = "Fix XYZ Glitch"
-
-# By convention, we call the options dataclass `<world>Options`.
-# It has to be derived from 'PerGameCommonOptions'.
-@dataclass
-class MyGameOptions(PerGameCommonOptions):
-    difficulty: Difficulty
-    final_boss_hp: FinalBossHP
-    fix_xyz_glitch: FixXYZGlitch
-```
-
-```python
-# __init__.py
-
-from worlds.AutoWorld import World
-from .options import MyGameOptions  # import the options dataclass
-
-
-class MyGameWorld(World):
-    # ...
-    options_dataclass = MyGameOptions  # assign the options dataclass to the world
-    options: MyGameOptions  # typing for option results
-    # ...
-```
```

Comment:
Is this going to be found somewhere else? I personally find examples to be really useful so removing them might not be a net positive.

Reply:
thought these were in the options api doc already. moved them now.